runtime/src/lib.rs

```diff
@@ -218,7 +219,7 @@ where
 					RawOrigin::Signed(caller).into(),
 					recipient_account,
 					transfer_amount.into(),
-				);
+				).map_err(|dispatch_result| dispatch_result.error)?;
```

Comment:
Just FYI: This is a [`DispatchErrorWithPostInfo`](https://github.com/paritytech/substrate/blob/6d09a45caf40a143791deab4f605b95230b787d1/primitives/runtime/src/lib.rs#L506), not a result.